src/Symfony/Component/HttpClient/Chunk/DataChunk.php

```diff
@@ -20,8 +20,8 @@
  */
 class DataChunk implements ChunkInterface
 {
-    private $offset;
-    private $content;
+    private $offset = 0;
+    private $content = '';
```

Comment:
what's the benefit of adding these when they are in the constructor?

Reply:
not having to call the constructor in child classes if defaults are appropriate, see `InformationalChunk`

Reply:
makes sense :+1: